internal/kafka/internal/handlers/validation.go

```diff
@@ -36,14 +35,29 @@ var ClusterIdLength = 32
 
 const minimunNumberOfNodesForTheKafkaMachinePool = 3
 
-func ValidateBillingModel(kafkaRequestPayload *public.KafkaRequestPayload) handlers.Validate {
+func validateKafkaBillingModel(ctx context.Context, kafkaService services.KafkaService, kafkaConfig *config.KafkaConfig, kafkaRequestPayload *public.KafkaRequestPayload) handlers.Validate {
```

Comment:
I renamed the method to `validateKafkaBillingModel` to be more specific about what we validate in here

Reply:
I also made it unexported as there is no need to expose it outside of the package